src/main/main.ino

```diff
@@ -98,3 +103,53 @@ void blink() {
     currentEffect = (currentEffect + 1) % currentAmountOfEffects;
   }
 }
+
+void changeEffect(int result) {
+    switch (result) {
+      case 0x97483BFB: //Keypad button "0"
+        Serial.println("Testeffect");
+        test();
+        break;
+      case 0xE318261B: //Keypad button "1"
+        Serial.println("Firework");
+        firework();
+        break;
+      case 0x511DBB: //Keypad button "2"
+        Serial.println("GOL");
+        gameOfLifeAnimation();
+        break;
+      case 0xEE886D7F:  //Keypad button "3"
+        Serial.println("Clock");
+        clockAnimation();
+        break;
+      case 0x52A3D41F: //Keypad button "4"
+        Serial.println("Stars");
+        starAnimation();
+        break;
+      case 0xD7E84B1B: //Keypad button "5"
+        Serial.println("Temperature effects");
+        tempSensorInfo();
+        break;
+      default:
+        Serial.println("Default");
+        break;
+    }
+  }
```

Comment:
Does this work for any keypad or did you use a specific one?

Reply:
It technically works for any infra-red emitting control, however, the hex codes differ from device to device. It is currently only implemented for the device I have got with the Arduino Mega set, however, it is easily scalable.